### PR TITLE
Popup window feature

### PR DESCRIPTION
**Pull request by lucas-crodrigues**
Add popup feature to the projects section

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -14,3 +14,131 @@ close.addEventListener('click', () => {
...
+
+  Object.keys(projects).forEach((project) => {
+    if (event.target.id === projects[project].id.toString()) {
+      selectedProjects = projects[project];
+    }
+  });
+
+  popupProjectMobile.innerHTML = `
+  <div class="popup-top" id="popup-top">
+    <h4 class="popup-header" id="popupHeader">${selectedProjects.projectTitle}</h4>
```
Comment (ShoiraTa) [inline scripts.js:113]: - [ ] In mobile view (starts at 425 px), the bottom of popup modal is not visible and the buttons seem to get out of the popup modal bottom border, kindly address this issue:
Screenshot 2022-04-12 at 17 03 50

- [ ] Please make sure that the content inside buttons does not wrap and is responsive.

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
STATUS: CHANGES REQUIRED 🟡 
----
Hi team,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!

woohoo-youre-almost


 ## Required Changes  :recycle: 

- [ ] Please make sure that all participants equally contribute to the project. Out of your commit history I can see that @lucas-crodrigues has the most number of commits and  Gedewon Haile has only two. 
- [ ] Please update your Readme, currently some headings are missing such as  

```
## Built With
## Authors
```
I would also recommend to add the following: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Microverse
```

_Check the comments under the review._ 


***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -66,6 +66,7 @@ <h3 class="headlineh2">I am a Software Developer</h3>
             </li>
         </ul>
     </section>
+    <div class="project-popup-mobile popup"></div>
     <section class="works" id="works">
         <div class="works-header">
```
Comment (BenMukebo) [inline index.html:71]: https://github.com/lucas-crodrigues/lucas-crodrigues.github.io/blob/28da9e77263dea377709775a3bf9fb7ce5589e24/index.html#L75-L245

- [ OPTIONAL] In this particular project, the whole projects section along with the modals should be created dynamically with Javascript as you did with you modals popup.
- The modals  is already done (good job!), but the works projects section are still not being created with Javascript.  Kindly please use Javascript to dynamically render your works projects section.

You could reuse the code that you already have.

- That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `data` array.

image

**Review by BenMukebo — APPROVED** [review]
### Hi @lucas-crodrigues , 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [OPTIONAL] Kindly, Add a descriptive README file to your project, it will be great to add instructions to set the project up locally  This includes `git clone url` , `cd project-name` etc

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**